### ISSUE TITLE
fix recursion problem in execution log viewer

### ISSUE THREAD

**Issue by franzlst** [DLR-RM/RAFCON#428]
File "/internal_path/common/packages/rafcon/0.11.5/bin/execution_log_viewer.py", line 29, in add_collapsed_key
    self.add_collapsed_key(parent, self.next_[key])
  File "/internal_path/common/packages/rafcon/0.11.5/bin/execution_log_viewer.py", line 26, in add_collapsed_key
    piter = self.treestore.append(parent, ["%s (%s)" % (self.items[key]['state_name'],self.items[key]['state_type']), str(key)])
RuntimeError: maximum recursion depth exceeded while calling a Python object

Originally created by SebastianRiedela  (Sebastian.Riedel@dlr.de) at 2017-10-19 08:54:34+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
Ok that is a thing I programmed. What was the situation in which it occurs and can you reproduce it?

Originally created by @Rbelder  at 2017-10-19 10:04:23+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
Ok not programmed by me 😄  ... miss the part that says `log`. But looks a lot like a failure I have had in the execution history widget.

Originally created by @Rbelder  at 2017-10-19 10:09:31+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
Just for clarification: I have reported that issue to @SebastianRiedela, it is a problem within the `execution_log_viewer.py`. He said he will fix it, this it why it is also assigned to him. The issue occurs, if the viewer should open an execution log with too many entries.

There has been a similar issue in the execution history widget, but this has already been fixed some time ago.

Originally created by @franzlst  (Franz.Steinmetz@dlr.de) at 2017-10-19 10:37:36+00:00 (moved from RMC internal repository)
